lib/rmg/Zone.cpp

```diff
@@ -193,11 +193,7 @@ void Zone::fractalize()
 	rmg::Area clearedTiles(dAreaFree);
 	rmg::Area possibleTiles(dAreaPossible);
 	rmg::Area tilesToIgnore; //will be erased in this iteration
-	
-	//the more treasure density, the greater distance between paths. Scaling is experimental.
-	int totalDensity = 0;
```

Comment:
Lets wait for @Nordsoft91 if this was intended

Reply:
It inherited from previous rmg, can be removed safely